README.md

```diff
@@ -40,109 +116,222 @@ You can download the script from the following locations:
-software to generate maps of your worlds.  Minecraft Overviewer is a
-command-line tool for rendering high-resolution maps of Minecraft worlds. It
-generates a set of static html and image files and uses the Google Maps API to
-display a nice interactive map.
+3. Drag the folder of the world you wish to move into the folder you just created. So I would drag the world `world` into the `vanillaMC` folder. The path of `world` (the actual world folder) would now be `/opt/mscs/worlds/vanillaMC/world/`.
 
-You can [download](http://overviewer.org/downloads) premade binaries for
-supported systems, or build your own binary from source if needed.
+The finished file structure should resemble the following, with a world named `world` and a containing folder name "vanillaMC":
+````
+/opt/mscs/vanillaMC       // The path
 
-Repositories for automatic installation are also available:
-* [Debian/Ubuntu](http://overviewer.org/debian/info)
-* [RHEL/CentOS/Fedora](http://overviewer.org/rpms/info)
+world                     // The actual world folder
+server.properties         
+other world files... (in no particular order)
+````
 
-### Firewall / NAT
-If you have a firewall installed on your computer, or a router using NAT
-installed in your network, you will need to route some ports to your server.
-Instructions on how to accomplish this are beyond the scope of this post, but
-here are some things you will need to know:
-* The default port for the Minecraft server is: `25565`.
-* If you wish to run multiple world servers using this script, you will
-  want to open a range of ports (for example `25565 - 25575`).
+After you've set up the file structure, you now need to create a world entry into MSCS. Do this via:
 
-See the [iptables.rules](iptables.rules)
-file for a very basic set of rules that you can use with the Iptables firewall.
+        mscs create [world] [port] <ip>
+  
+Where `world` is the **name of the containing folder you created** (so it would be "vanillaMC" from the previous example", and `port` is the server port (by default, use 25565).
+`ip` is optional and will be used if you wish to create multiple worlds across different servers. For now, leave it blank.
+
+Afterwards, simply start the server via `mscs start [world]` where `world` is the name of the containing world's folder (again, it would be "vanillaMC" from the last example). Then, accept the EULA, run the start command again, and you're good to go!
+
+As a last note, make sure you check `server-port` and `query.port` in `server.properties` to make sure it does not overlap with other servers created by the MSCS script. Also ensure that `enable-query` is set to `true`.  If you do not have `enable-query` and a `query.port` set, you will not be able to check the status of the world with the script.
+
+
+## Adjusting server options
+There are two ways of adjusting the options through MSCS: changing values in the mscs.properties file and/or editing the msctl file directly.
+
```

Comment:
Lets not advocate editing the script files directly.  Use /etc/default/mscs or one of the alternatives instead.